### PR TITLE
Fixes for CVE`s titles and for missing packages in bulletins

### PR DESCRIPTION
**Pull request by anfimovdm**

### CODE REVIEW

File: scripts/errata_fix_script.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+async def update_albs_db():
+    logging.info("Update albs db started")
+    query = (
+        select(ErrataReference)
+        .where(
+            ErrataReference.ref_type != ErrataReferenceType.bugzilla,
+            ErrataReference.title == '',
+        )
+    )
+    async for db in get_db():
```
Comment (Korulag) [inline scripts/errata_fix_script.py:182]: you can use `contextlib.asynccontextmanager(get_db)()` construct, this way you'll be able to write whole `with` statement in one string

**Review by soksanichenko — APPROVED** [review]
will lgtm when you fix the Vasily's comment

File: scripts/errata_fix_script.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                pulp_pkg = albs_packages_cache[filename]
+                if (
+                    pulp_pkg['sha256'] is None
+                    and pulp_pkg['checksum_type'] == 'sha256'
+                ):
+                    logging.warning(
+                        'sha256 sum is missing for %s using pkgId instead',
+                        filename,
+                    )
+                    pulp_pkg['sha256'] = pulp_pkg['pkgId']
```
Comment (javihernandez) [inline scripts/errata_fix_script.py:133]: Maybe stupid question. Is this the right thing to do here? Why not just leaving it empty?

**Review by javihernandez — COMMENTED** [review]
LGTM and will definitely approve it when comments/questions are addressed/responded. Great job!!

File: scripts/errata_fix_script.py (hunk 2 above)
Comment (anfimovdm) [reply scripts/errata_fix_script.py:133]: this field have not null constraint in pulp DB

**Review by javihernandez — APPROVED** [review]

**Review by Korulag — APPROVED** [review]

**Pull request merged**